ranger-discovery-bundle/src/main/java/io/appform/ranger/discovery/bundle/id/formatter/IdParsers.java

```diff
@@ -44,7 +50,18 @@ public Optional<Id> parse(final String idString) {
             if (!matcher.find()) {
                 return Optional.empty();
             }
-            return IdFormatters.original().parse(idString);
+
+            val parserType = matcher.group(3);
```

Comment:
should we put a guard condition check on length of matched parserType. Ideally it should only be 2 chars

Reply:
Since pattern specifies only matching integers of 2 digits `([0-9]{2})?`. Any extra digit in parserType would be read as suffix.